### ISSUE TITLE
Corrected long reads?

### ISSUE THREAD

**Issue by andersgs** [rrwick/Unicycler#4]
Hi Ryan.

I suspect it doesn't matter, but I was just asked this: does `unicycler` use raw or corrected long-reads (say correcting with `canu`, for instance)? (I also suspect someone else might ask you this, so now it is documented).

The reason I think it does not matter is that it is used only for threading the graph. Some mismatch may occur, and that will be fine, and seems like it can be controlled by the alignment score option. Do you have any tests showing any differences in the output depending on using `raw` or `corrected` reads? 

Cheers.

A.

**Comment by rrwick** [comment]
Unicycler doesn't explicitly correct all of the reads you give it. But if it's using multiple reads to bridge two contigs in the graph, it does an MSA and then makes a consensus read sequence (just for the relevant parts of the reads). So in a sense it does error-correct them internally when it uses them.

I haven't done detailed tests on corrected vs uncorrected reads, but my hunch is that it would perform better with uncorrected reads. But it may depend on how they are corrected. If the correction process reduces read length and/or read depth, then it will probably negatively impact Unicycler's performance.

**TL;DR:** use raw reads.
